### PR TITLE
Added Save and Load functions

### PR DESCRIPTION
**Pull request by lowrykun**
I added 4 fields:
1. Save As Field
2. Save button
3. Load File field
4. Load button

Using these you can save your current console to a text file and load your console from a text file.

### CODE REVIEW

File: php-console.js [hunk 1]
```diff
@@ -181,6 +181,54 @@
...
+        //Load
+        if (window.localStorage) {
+            $('.statusbar .load').on('click', function (e) {
+            	var fileToLoad = document.getElementById("fileToLoad").files[0];
+
+            	var fileReader = new FileReader();
+            	fileReader.onload = function(fileLoadedEvent) 
+            	{
+            		var textFromFileLoaded = fileLoadedEvent.target.result;
+            		//document.getElementById("inputTextToSave").value = textFromFileLoaded;
```
Comment (staabm) [inline php-console.js]: Please remove commented code

File: styles.css [hunk 2]
```diff
@@ -31,6 +31,13 @@ input {
     font-size: 2em;
 }
 
+.f {
```
Comment (staabm) [inline styles.css]: Please use a more descriptive className

File: php-console.js [hunk 3]
```diff
@@ -181,6 +181,54 @@
...
+            	downloadLink.innerHTML = "Download File";
+            	if (window.webkitURL != null)
+            	{
+            		// Chrome allows the link to be clicked
+            		// without actually adding it to the DOM.
+            		downloadLink.href = window.webkitURL.createObjectURL(textFileAsBlob);
+            	}
+            	else
+            	{
+            		// Firefox requires the link to be added to the DOM
```
Comment (staabm) [inline php-console.js]: Will this FF code also work for chrome?

Comment (lowrykun) [inline php-console.js]: Dunno. Copied it from here:
https://thiscouldbebetter.wordpress.com/2012/12/18/loading-editing-and-saving-a-text-file-in-html5-using-javascrip/

**Comment by lowrykun** [comment]
I made some of the changes you suggested.

File: php-console.js [hunk 4]
```diff
@@ -181,6 +181,53 @@
                 e.preventDefault();
             });
         }
+        
+        //Save
+        if (window.localStorage) {
```
Comment (staabm) [inline php-console.js]: This features are only available when localStorage is supported, but the links within the ui do not depend on this feature detect?

File: php-console.js (hunk 3 above)
Comment (staabm) [inline php-console.js]: Since you copied the code you need to give credit to the origin author

File: php-console.js (hunk 4 above)
Comment (Seldaek) [inline php-console.js]: I think we can rely on localStorage being there these days.. especially for such a dev oriented tool.

File: php-console.js (hunk 3 above)
Comment (Seldaek) [inline php-console.js]: A better approach would be to check if either of those code paths works for both browsers IMO, 3year old advice isn't necessarily relevant anymore.

File: php-console.js (hunk 4 above)
Comment (staabm) [inline php-console.js]: Ok, then we should also remove the feature detect a few lines below

File: php-console.js (hunk 3 above)
Comment (staabm) [inline php-console.js]: Rights, that was the intention of my initial question :-)

**Pull request closed without merging**